identity/src/main/java/com/stripe/android/identity/ui/SelfieScreen.kt

```diff
@@ -186,6 +186,12 @@ internal fun SelfieScanScreen(
             identityViewModel.resetSelfieUploadedState()
         }
 
+        LaunchedEffect(newDisplayState) {
+            if (newDisplayState is IdentityScanState.Finished) {
+                identityScanViewModel.stopScan(lifecycleOwner)
+            }
+        }
```

Comment:
seems like the state lives already on `identityScanViewModel`. Why do we need to observe it in the view, then call it in the viewmodel again? seems like it's not necessary to involve the view here.

Reply:
In order to start/stop camera, the`startScan` and `stopScan` calls API from `camera-core`, which bounds/unbounds a lifecyleOwner. If we call it from the viewModel there's no way to pass the lifecycle owner there